src/index.md

```diff
@@ -25,6 +25,10 @@
   * [Merging](producingChangeset.html#merging)
   * [Pushing](producingChangeset.html#pushing)
 * [What Happens Next](next.html)
+  * [Requesting a Backport](next.html#requesting-a-backport)
+  * [Filing a Bug](next.html#filing-a-bug)
+  * [ProblemListing or Ignoring a test](next.html#problemlisting-or-ignore-ing-a-test)
```

Comment:
shouldn't it use the exact same words as the section's title?
```suggestion
  * [ProblemListing or `@ignore`-ing a Test](next.html#problemlisting-or-ignore-ing-a-test)
```

Reply:
Good catch!